src/MOI_wrapper/MOI_wrapper.jl

```diff
@@ -94,11 +94,18 @@ MOI.supports(::Optimizer, param::MOI.RawParameter) = true
 
 function MOI.set(model::Optimizer, param::MOI.RawParameter, value)
     key = Symbol(param.name)
+    
+    # Setters for `SolveType` and `PresolveType` are handled separately,
+    # as they have a third `extraInfo` argument
     if haskey(CLP_OPTION_MAP, key)
         push!(model.options_set, key)
         CLP_OPTION_MAP[key][2](model.inner, value)
-    elseif haskey(SOLVE_OPTION_MAP, key)
-        SOLVE_OPTION_MAP[key][2](model.solver_options, value)
+    elseif key == :SolveType
+            ClpSolve_setSolveType(model.solver_options, value, -1)
+    elseif key == :PresolveType
+        ClpSolve_setPresolveType(model.solver_options, value, -1)
+    elseif key == :InfeasibleReturn
```

Comment:
I would have expected `elseif haskey(SOLVE_OPTION_MAP, key)` to be consistent with the comment `Setters for `SolveType` and `PresolveType` are handled separately`. Either update this line or the comment.

Reply:
Comment updated. I also fixed the indent.